astroquery/heasarc/core.py

```diff
@@ -206,6 +207,16 @@ def _parse_result(self, response, verbose=False):
         # if verbose is False then suppress any VOTable related warnings
         if not verbose:
             commons.suppress_vo_warnings()
+
+        if "BATCH_RETRIEVAL_MSG ERROR:" in str(response.content):
```

Comment:
could just do `response.text` instead of `str(response.content)`

Reply:
Will do. Also, will now use this for my deployment skipping test.